types/algoliasearch/index.d.ts

```diff
@@ -1778,12 +1779,20 @@ declare namespace algoliasearch {
     };
     /**
      * List of attributes on which to do a decomposition of camel case words.
-     *
-     https://www.algolia.com/doc/api-reference/api-parameters/camelCaseAttributes/
+     * https://www.algolia.com/doc/api-reference/api-parameters/camelCaseAttributes/
      */
     camelCaseAttributes?: string[];
-
+    /**
+     * Controls how facet values are sorted.
+     * https://www.algolia.com/doc/api-reference/api-parameters/sortFacetValuesBy/
+     */
     sortFacetValuesBy?: 'count' | 'alpha';
+    /**
+     * Sets the languages to be used by language-specific settings and functionalities
+     * such as ignorePlurals, removeStopWords, and CJK word-detection.
+     * https://www.algolia.com/doc/api-reference/api-parameters/queryLanguages/
+     */
+    queryLanguages?: string[];
```

Comment:
There's a list of accepted language codes. In a later iteration, or here, we can add those instead of "string": https://www.algolia.com/doc/api-reference/api-parameters/queryLanguages/?language=php#usage-notes

Reply:
Sounds like a good idea. I guess this list of supported languages doesn't change that often, does it? In that case, we might need to keep our list in-sync with the one from Algolia. Or we allow `'af'|'ar'|...|string` as a fallback here. What do you think?

Reply:
It won't change often, and is a place where it is easy fix if anything changes, so I don't think `string` is necessary as fallback

Reply:
Ok let me add the list of possible language codes then :)